### PR TITLE
[#172724206] upgrade dependencies

### PR DESCRIPTION
**Pull request by balanza**
Upgrade of `io-functions-commons` from v4 to v9.

Consequently, custom telemetry implementation has been removed

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "prestart": "npm run build && func extensions install",
     "start:host": "func start",
     "start": "npm-run-all --parallel start:host watch",
-    "pretest": "yarn generate:definitions",
+    "_pretest": "yarn generate:definitions",
```
Comment (gunzip) [inline package.json:12]: why this change here ?

Comment (balanza) [reply package.json]: just a typo..
